lib/modem_info/modem_info.c

```diff
@@ -53,6 +57,9 @@ LOG_MODULE_REGISTER(modem_info);
 #define CELLID_PARAM_INDEX 3
 #define CELLID_PARAM_COUNT 5
 
+#define AREA_CODE_PARAM_INDEX 2
```

Comment:
Nit: These values could be aligned with the above macros as well

Reply:
Fixed.